specs/engine.spec/src/main/resources/META-INF/zilla/core.idl

```diff
@@ -100,4 +100,36 @@ scope core
             octets extension;
         }
     }
+
+    scope event
+    {
+        enum Level
+        {
+            INFO,
+            WARNING,
+            ERROR
+        }
+
+        enum Result
+        {
+            SUCCESS,
+            FAILURE
+        }
+
+        struct Event
+        {
+            Result result;
```

Comment:
I suspect that `SUCCESS` / `FAILURE` is not generic enough to apply to all logged events.
Instead we can follow the pattern of having an extension on the `BindingEvent` that is `http` specific etc, matching the approach we've taken elswhere.